modelservice/src/main/java/com/expedia/adaptivealerting/modelservice/web/DetectorController.java

```diff
@@ -51,9 +51,9 @@ public String createDetector(@Valid @RequestBody DetectorDocument document) {
 
     @GetMapping(path = "/findByUuid", produces = "application/json")
     @ResponseStatus(HttpStatus.OK)
-    public DetectorDocument findByUuid(@RequestParam String uuid) {
-        DetectorDocument detector = detectorRepo.findByUuid(uuid);
-        if (detector == null) {
+    public List<DetectorDocument> findByUuid(@RequestParam String uuid) {
```

Comment:
This is a breaking change for clients. I think we're better off returning a single detector from `detectorRepo.findByUuid(uuid);`